ui/app/src/utils/object.ts

```diff
@@ -61,14 +61,24 @@ export function createLookupTable<T>(list: T[], idProp: string = 'id'): LookupTa
   return table;
 }
 
-export function createNodesLookup(list: any[], idProp: string = 'id'): LookupTable {
-  const table = {};
-  list.forEach((item) => {
-    table[retrieveProperty(item as any, idProp)] = item;
-    item.children?.forEach(children => {
-      table[retrieveProperty(children as any, 'id')] = children;
+export function createNodesLookup<T>(list: T[], idProp: string = 'id'): LookupTable<T> {
```

Comment:
Revert this whole file. Please pull from my fork and use the new hierarchicalToLookupTable object util.